src/main/java/random_csapatnev/MainFrame.java

```diff
@@ -14,13 +14,16 @@
  *
  */
 public class MainFrame extends JFrame {
+	Random rand = new Random();
+	private static String fontstr = "Arial";
+
 	/**
 	 * 
 	 */
 	private static final long serialVersionUID = 1L;
```

Comment:
Ez is átmozgatható lehet a többi konstanssal.

Reply:
[c380e91](https://github.com/BME-MIT-IET/iet-hf-2023-grafadatbazis/pull/14/commits/c380e9172c125819a458b7c3964a396f8d0b03ce) commitban javítva.